src/index.html

```diff
@@ -7,8 +7,68 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#" class="container__image">
+        <img
+          src="images/logo.png" 
+          alt="image"
+        >
+      </a>
+      <nav class="container__nav">
```

Comment:
```suggestion
      <nav class="nav">
```

Reply:
Use BEM please
`container__nav` mean that you should have parent block with class container and nav is element from this container